server.go

```diff
@@ -62,6 +62,7 @@ func (s *server) Start() (bool, error) {
 	}, bufSize)
 
 	<-s.persister.ConfigChangedWatcher()
+	s.createBucketContainer()
```

Comment:
Do you mean every time after `<-s.persister.ConfigChangedWatcher()` and not just here?